GetAssetTrigger/popularasset.ts

```diff
@@ -0,0 +1,72 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+import axios, { AxiosError } from "axios"
+import { config } from "../config"
+
+const GetPopularAssets: AzureFunction = async function (context: Context, req: HttpRequest): Promise<string> {
+  const limit = Number.isNaN(Number(req.query.limit)) ? undefined : Number(req.query.limit)
+  const offset = 0
+  if (!limit) {
+    console.log("[GetPopularAsset] Invalid limit param", req.query.limit)
+    context.res = {
+      status: 400,
+    }
+  } else {
+    try {
+      const { data: approvedStatus } = await axios.get<any>(`${config.COLLIBRA_BASE_URL}/statuses/name/Approved`, {
```

Comment:
i've added the collibra type defs we have in data marketplace to this project, so you can use them here.